source/2020-10-01-hacktoberfest.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Hacktoberfest 2020
+author: Jen Weber
+tags: Recent Posts, Announcement
+responsive: true
+---
+
+Have you ever spotted something in Ember code or documentation that needed fixing?
+Did you want to get involved in contributing to open source, but you have no idea where to start?
+
+Good news! It's Hacktoberfest 2020, and you are invited to participate in Ember-focused special events and mentorship opportunities through the month of October.
+You can make some commits, meet some new people, and maybe get some free stuff!
+
+## What is Hacktoberfest?
+
+[Hacktoberfest](https://hacktoberfest.digitalocean.com/) is an annual, worldwide celebration organized by DigitalOcean and sponsored by many companies that use open source software.
+
+The premise is simple: 
+- Sign up anytime between October 1 and October 31
+- Make four valid Pull Requests to _any_ public repositories on GitHub
+- If you are among the first 70,000 developers who complete the challenge, you get a prize. This year's prizes include T-shirts or a tree planted in your name.
+
+Many open source project maintainers put in some extra effort during Hacktoberfest to make it easy and fun to participate in open source.
+
+## What is happening in the Ember community?
+
+Here's what you can expect this year!
+
+- **On Saturday, October 10th,** past contributors will be available to pair with anyone who wants to help out! If you have questions or need some help finding an issue to work on, this is a great place to start. Specific times will be announced soon.
+- **On Friday, October 16th,** there will be an Ember Contributors Workshop! DESCRIPTION AND LINK
+- **Any time,** join the `#hacktoberfest` channel in the [Ember Community Discord]() to connect with other developers. You can find a buddy to pair with, a mentor to guide you, or toss around ideas.
+
+If you are someone who is willing to help mentor new contributors, please let us know in `#hacktoberfest` on Discord. Addon maintainers are welcome to post issues they need help with too!
+
+Additionally, there are several more groups within the Ember Community Discord where you can find others to collaborate with. There are Discord channels for women, lgbtq+, and people of color. If you belong to one of those groups, you can ask to join in the `#discord-server-admin` channel.
```

Comment:
```suggestion
Additionally, there are several more groups within the Ember Community Discord where you can find others to collaborate with. There are Discord channels for women, LGBTQ+, and people of color. If you belong to one of those groups, you can ask to join in the `#discord-server-admin` channel.
```
In addition to asking publicly in the admin channel, can we provide people the option to DM a mod (maybe Melanie)?